qiskit_experiments/visualization/plotters/curve_plotter.py

```diff
@@ -64,7 +66,8 @@ def expected_supplementary_data_keys(cls) -> List[str]:
                 ``textbox_text_size`` style parameters in :class:`PlotStyle`.
         """
         return [
-            "report_text",
+            "primary_results",
```

Comment:
I'm happy with the names of the new supplementary data-keys. Could you update the docstring of `expected_supplementary_data_keys` to reflect the new data-keys?